metadata/io.undertow/undertow-core/2.3.0.Final/resource-config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "resources":{
+  "includes":[{
+    "condition":{"typeReachable":"io.undertow.attribute.ExchangeAttributeParser"},
+    "pattern":"\\QMETA-INF/services/io.undertow.attribute.ExchangeAttributeBuilder\\E"
```

Comment:
all the `META-INF/services` can be removed. This is handled automatically in GraalVM.